com.unity.netcode.gameobjects/Runtime/Messaging/RpcTargets/RpcTarget.cs

```diff
@@ -67,9 +67,28 @@ public enum SendTo
         SpecifiedInParams
     }
 
+    /// <summary>
+    /// This parameter controls the use of a particular performance optimization in situations where it can be used.<br />
```

Comment:
Not sure whether this is a field the client can set themselves, or whether we set it for them (i.e. information about the RpcTarget that NGO populates. If this is to be set by the client, I think the wording can be clearer. 

e.g:
```suggestion
    /// This parameter configures a performance optimization. This optimization is not valid in all situations.<br />
```